### DISCUSSION TITLE
Editing a previously created image

### DISCUSSION THREAD

**Discussion by sharjeelnisar2** [personalizedrefrigerator/js-draw#76] [General]
Hi,
   Do we have an API to open an image, previously drawn via draw-js, in edit mode?

**Comment by personalizedrefrigerator** [comment]
Hi!

The [`.loadFromSVG`](https://personalizedrefrigerator.github.io/js-draw/typedoc/classes/js_draw.Editor.html#loadFromSVG) and [`.toSVG`](https://personalizedrefrigerator.github.io/js-draw/typedoc/classes/js_draw.Editor.html#toSVG) APIs can be used to load and save images ([example](https://personalizedrefrigerator.github.io/js-draw/typedoc/modules/js_draw.html) that uses both of these APIs).

The saved SVG can then be saved using `localStorage`, `indexedDB`, or another API (the [demo app](https://github.com/personalizedrefrigerator/js-draw/blob/main/docs/demo/index.ts) uses `indexedDB`).

Please let me know if you have any follow-up questions!
